sphinx_gallery/notebook.py

```diff
@@ -125,16 +134,53 @@ def rst2md(text, heading_levels):
     text = re.sub(contents, '', text)
 
     images = re.compile(
-        r'^\.\. image::(.*$)(?:\n *:alt:(.*$)\n)?(?: +:\S+:.*$\n)*',
+        r'^\.\. image::(.*$)((?:\n +:\S+:.*$)*)\n',
         flags=re.M)
+    image_opts = re.compile(r'\n +:(\S+): +(.*)$', flags=re.M)
     text = re.sub(
-        images, lambda match: '![{1}]({0})\n'.format(
-            match.group(1).strip(), (match.group(2) or '').strip()), text)
+        images,
+        lambda match: '<img src="{}"{}>\n'.format(
+            generate_image_src(
+                match.group(1).strip(), gallery_conf, target_dir),
+            re.sub(image_opts, r' \1="\2"', match.group(2) or '')),
+        text)
 
     return text
 
 
-def jupyter_notebook(script_blocks, gallery_conf):
+def generate_image_src(image_path, gallery_conf, target_dir):
+    if re.match(r'https?://', image_path):
+        return image_path
+
+    if not gallery_conf['notebook_images']:
+        return "file://" + image_path.lstrip('/')
+
+    if image_path.startswith('/'):
+        # Absolute path from source directory
+        target_dir = gallery_conf['src_dir']
+        image_path = image_path.lstrip('/')
+    full_path = os.path.join(target_dir, image_path.replace('/', os.sep))
```

Comment:
I might be confused here but is it possible to refer to image file but not start with '/', with rST `image` directive? 
From [sphinx docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#images), you can refer to image using:
```rst
.. image:: gnu.png
or
.. image:: ../images/gnu.png
```

Reply:
Yes. So if it's absolute (starting with `/`), full path is relative from the documentation source directory. If it's relative (not starting with `/`), full path will be from `target_dir` (location of RST).

Reply:
Ah yes, thank you, I was confused.

For other poor readers could we move the comment:
```
    # If absolute path from source directory given
    if image_path.startswith('/'):
        
```